tahoe_idp/templates/tahoe_idp/magiclink_login_failed.html

```diff
@@ -0,0 +1,38 @@
+<!doctype html>
```

Comment:
Let's use stock `/500` instead.

Reply:
the view is returning 500 error now (not redirecting to tahoe's 500). I'd like to keep it like this and change it in the next PR if needed. What do you say @OmarIthawi ?